webtau-http/src/main/java/org/testingisdocumenting/webtau/http/datanode/DataNodeCompareToHandler.java

```diff
@@ -41,21 +48,33 @@ public boolean handleGreaterLessEqual(Object actual, Object expected) {
 
     @Override
     public void compareEqualOnly(CompareToComparator comparator, ActualPath actualPath, Object actual, Object expected) {
+        DataNode actualDataNode = (DataNode) actual;
+        CompareToResult result;
         if (expected instanceof Map) {
-            compareWithMap(comparator, actualPath, (DataNode) actual, (Map) expected);
+            result = compareWithMap(comparator, actualPath, actualDataNode, (Map) expected);
         } else {
-            Object extractedActual = extractActual((DataNode) actual);
-            comparator.compareUsingEqualOnly(actualPath, extractedActual, expected);
+            Object extractedActual = extractActual(actualDataNode);
+            result = comparator.compareUsingEqualOnly(actualPath, extractedActual, expected);
+        }
+
+        if (actualDataNode.isNull() || actualDataNode.get() == null) {
```

Comment:
what if actual is not directly null, but something like 
```
person.should == [address: [street: 'name']
```
where `address` is not present on actual `person`